CONTRIBUTING.md

```diff
@@ -275,3 +275,22 @@ see <https://rustc-dev-guide.rust-lang.org/building/how-to-build-and-run.html>.
 
 With this, you should now have a working development setup! See
 [above](#building-and-testing-miri) for how to proceed working on Miri.
+
+### Syncing the miri subrree
+
+#### Updating from the rustc repo
+
+One time setup:
+
+Add your own miri fork as `miri-your-fork` to the rustc repo:
+
+```
+git remote add miri-your-fork git@github.com:your-name/miri.git
+```
+
+Every time:
+
+```
+ulimit -s 60000
```

Comment:
Why do we need to change the stack size to run this?

Reply:
because the script will segfault bash otherwise 😫 See also https://github.com/rust-lang/rust-clippy/issues/5565 where all of this was figured out

Reply:
Can't the script do this itself?

Reply:
I didn't want to modify it ^^